adapters/Snap/SnapAdapter/GADMediationSnapBanner.m

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "GADMediationSnapBanner.h"
+
+#import "GADMediationAdapterSnapConstants.h"
+
+#import <SAKSDK/SAKSDK.h>
+
+static SAKAdViewFormat _GADSAKAdViewFormatFromAdSize(GADAdSize adSize, BOOL *invalid)
```

Comment:
Should use `GADClosestValidSizeForAdSizes` to check the ad size in this method.

See below for reference: 
https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/Nend/NendAdapter/GADMAdapterNend.m#L24-L36